### PR TITLE
Take workflow configuration file from external URL

### PR DESCRIPTION
**Pull request by saraycp**
From now on, the workflows configuration file doesn't need to be inside the repository. We can specify any other accessible URL.

Screenshot 2022-12-20 at 09-18-44 Open Build Service

### CODE REVIEW

File: src/api/app/views/webui/users/tokens/edit.html.haml [hunk 1]
```diff
@@ -49,12 +49,18 @@
...
                   .form-text.text-muted
-                    The default path is '.obs/workflows.yml'
+                    The default path is '.obs/workflows.yml'.
+                .form-group#workflow-configuration-url-group
+                  = f.label(:workflow_configuration_url, 'URL to Workflows Configuration File:')
+                  .input-group
+                    = f.text_field(:workflow_configuration_url, class: 'form-control',
+                                                                placeholder: 'Eg: https://example.com/.my_subdir/my_workflows_file.yml')
+                  .form-text.text-muted
+                    When the URL is given, the path will be ignored. It should be a public URL.
```
Comment (dmarcoux) [inline src/api/app/views/webui/users/tokens/edit.html.haml:63]: I think we should rephrase this. The URL has to be accessible for the OBS instance, which might be public or not, this we don't know.

File: src/api/app/views/webui/users/tokens/show.html.haml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+-# haml-lint:disable ViewLength
```
Comment (dmarcoux) [inline src/api/app/views/webui/users/tokens/show.html.haml:1]: Refactoring this view is tracked in our backlog [here](https://trello.com/c/6Oe3vBnC/214-split-up-the-users-tokens-showhtmlhaml-view).

**Review by danidoni — APPROVED** [review]

**Comment by danidoni** [comment]
I would appreciate a screenshot with the new changes in the form

**Comment by saraycp** [comment]
> I would appreciate a screenshot with the new changes in the form

Screenshot and description added.

**Comment by dmarcoux** [comment]
I will handle the Brakeman false positive. It needs to be ignored.

**Review by dmarcoux — APPROVED** [review]

**Pull request merged**